docs/reference/esql/functions/operators.asciidoc

```diff
@@ -13,20 +13,16 @@ Boolean operators for comparing against one or multiple expressions.
 * <<esql-logical-operators>>
 * <<esql-predicates>>
 * <<esql-cidr_match>>
```

Comment:
Maybe we should move this one as well while we're at it? It's a function, not an operator.

Reply:
I was looking at it too, but didn't find a clearly better home for it? Maybe `Conditional functions`?

Reply:
I'll leave it as is in this PR. Will open a new one to add a new `IP functions` section, this doesn't properly belong to any existing ones, IMO.